### ISSUE TITLE
liberfa files missing in release and repository

### ISSUE THREAD

**Issue by Stefan-Heimersheim** [liberfa/pyerfa#80]
The `liberfa/erfa` directory is empty, both in the release archive and the git repository:
```
cd pyerfa-2.0.0.1/
ls liberfa/erfa
```
and `git submodule update` does not do anything for me.

One can clone the erfa repo manually and move the files into that directory -- I would suggest adding this to the documentation (maybe as a sequence of command lines) to help new users :)

Cheers,
Stefan

**Comment by avalentino** [comment]
Not clear what you mean by "release archive", the one the you can download form GitHub?

Regarding the git repository, I think that you should use
```shell
$ git clone --recursive ...
```
or you need
```shell
$ cd pyerfa
$ git submodule init
```
before
```shell
$ git submodule update
```

**Comment by Stefan-Heimersheim** [comment]
Thank you! The version you suggested works with the git repository.

This solves the problem for me -- I think adding this to the Installation instructions section would be great! Note though that this does not with the "release" shown on GitHub here: [https://github.com/liberfa/pyerfa/releases](https://github.com/liberfa/pyerfa/releases), MWE:
```
wget https://github.com/liberfa/pyerfa/archive/refs/tags/v2.0.0.1.tar.gz
tar xvf v2.0.0.1.tar.gz
cd pyerfa-2.0.0.1/
ls liberfa/erfa/
# empty, and also not a git repository
pip install .
```
`pip` fails with
```
  RuntimeError: Missing "liberfa" source files, unable to generate "erfa/ufunc.c" and "erfa/core.py". Please check your source tree. Maybe "git submodule update" could help.
```
(full traceback attached [log.txt](https://github.com/liberfa/pyerfa/files/7474317/log.txt)). But again, thank you for your help, I got it installed your way, the rest is just in case you want to fix/reproduce my problems with the "release archive" (tar.gz) file.

**Comment by avalentino** [comment]
@Stefan-Heimersheim now I see your point about the release archive.
Unfortunately I'm not sure how to fix it. Suggestions are welcome.
By the way the "official" source archive is the one PyPi.

Regarding the installation instructions in our readme file, you are welcome to submit a PR if you want.
Otherwise I will try to improve it as soon as I have the time.
In any vase thanks a lot for reporting.

**Comment by mhvk** [comment]
closing since fixed by #81.
